frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -81,11 +80,11 @@ export const DashboardHeaderInner = ({
     if (isDirty && isEditing) {
       setShowCancelWarning(true);
     } else {
-      onCancel();
+      onConfirmCancel();
     }
   };
 
-  const onCancel = () => {
+  const onConfirmCancel = () => {
```

Comment:
I renamed this because the action being confirmed is to 'Cancel', but it's confusing because the modal also has a Cancel button